skops/hub_utils/_hf_hub.py

```diff
@@ -327,6 +327,62 @@ def init(
         raise
 
 
+def add_files(*files: str | Path, dst: str | Path, exist_ok: bool = True) -> None:
+    """Add files to initialized repo.
+
+    After having called :func:`.hub_utils.init`, use this function to add
+    arbitrary files to be uploaded in addition to the model and model card.
+
+    In particular, it can be useful to upload the script itself that produces
+    those artifacts by calling ``hub_utils.add_files([__file__], dst=...)``.
+
+    Parameters
+    ----------
+    *files : str or Path
+        The files to be added. If a file by that name already exists at target
+        location, it will be skipped.
+
+    dst : str or Path
+        Path to the initialized repo, same as used during
+        :func:`.hub_utils.init`.
+
+    exist_ok : bool (default=True)
```

Comment:
Probably set the default to ``False``, to avoid accidental overwrite?

Reply:
I think I slightly prefer the default to be `True`. That way, calling it multiple times is fine (idempotency).

Reply:
Not really sure if there is a Python convention for `exist_ok`: https://docs.python.org/3/library/pathlib.html#pathlib.Path.touch

Anyway, I changed it, as mentioned it was only a slight preference.